app/controllers/playlists_controller.rb

```diff
@@ -15,6 +17,10 @@ def new
   def create
     @playlist = Playlist.new(playlist_params)
 
+    if @playlist.valid?
+      PlaylistCreator.new(@playlist.genre, @playlist.energy, @playlist.danceability, @playlist).get_playlist
+    end
```

Comment:
```suggestion
    if @playlist.valid?
      @playlist.spotify_id = PlaylistCreator.new(@playlist.genre, @playlist.energy, @playlist.danceability, @playlist).get_playlist
    end
```

C'est mieux que le service ne s'occupe pas de faire de modification sur le modèle, pour éviter que le service ait trop de responsabilité ( https://en.wikipedia.org/wiki/Single-responsibility_principle )